frontend/src/lib/api.ts

```diff
@@ -256,6 +265,28 @@ const api = {
         },
     },
 
+    activity: {
+        list(activityLogProps: ActivityLogProps, page: number = 1): Promise<CountedPaginatedResponse> {
+            const urlForScope: {
+                [key in ActivityScope]: (props: ActivityLogProps) => string
+            } = {
+                [ActivityScope.FEATURE_FLAG]: (props) => {
+                    return props.id
+                        ? `/api/projects/@current/feature_flags/${props.id}/activity`
+                        : `/api/projects/@current/feature_flags/activity`
```

Comment:
We should really avoid using `@current` in any new (and old) code, as otherwise you can't have two tabs open with two separate projects. You'll see activity from the last opened project on both tabs, and that's just weird. Can this be replaced with the right team ID?